### ISSUE TITLE
Need for Tutorial

### ISSUE THREAD

**Issue by irenemaria93** [ManuelPeinado/FadingActionBar#118]
Hello Team,
I loved the design layouts posted by your team.It would appreciate if you could include the steps to run the code in Android Studio SDK ! 
Thanks,
Irene

**Comment by silviahisham** [comment]
I agree, Please provide a simple tutorial or documenation for the methods used

**Comment by shixinzhang** [comment]
You gave me a reason to use the ADT again~
